src/System.Runtime.Extensions/src/System/Environment.Unix.cs

```diff
@@ -191,7 +191,7 @@ private static int FindAndParseNextNumber(string text, ref int pos)
             return num;
         }
 
-        private static int ProcessorCountCore => (int)Interop.Sys.SysConf(Interop.Sys.SysConfName._SC_NPROCESSORS_ONLN);
+        public static int ProcessorCount => (int)Interop.Sys.SysConf(Interop.Sys.SysConfName._SC_NPROCESSORS_ONLN);
```

Comment:
Thanks. Worth keeping in mind (or even opening an issue if you think there's actually a lurking problem here), but I won't change it as part of this PR, as this is all pre-existing (even prior to my moving this code to C#, as this is what coreclr's implementation of GetSystemInfo uses in its PAL).